### PR TITLE
release: v0.13.1

### PR DESCRIPTION
**Pull request by iGormilhit**
Co-Authored-by: Igor Milhit <igor.milhit@rero.ch>

## Why are you opening this PR?

For the v0.13.1 release

## How to test?

- Check the changed files, especially the release notes for v0.13.0 and v0.13.1 

## Code review check list

- [ ] Commit message template compliance.
- [ ] Commit message without typos.
- [ ] File names.
- [ ] Functions names.
- [ ] Functions docstrings.
- [ ] Unnecessary commited files?

### CODE REVIEW

File: RELEASE-NOTES.rst [hunk 1]
```diff
@@ -18,6 +18,293 @@
...
+~~~~~~~~~~~~~~
+
+-  Updates the help link of the homepage to the actual help instead of
+   the old GitHub wiki page.
+
+Circulation
+~~~~~~~~~~~
+
+-  Fixes ILL request form validation issues that prevents ILL request to
+   be saved, or that saves ILL request with wrong data.
```
Comment (mmo) [inline RELEASE-NOTES.rst:37]:
```suggestion
   be saved, or that saves ILL requests with wrong data.
```

File: RELEASE-NOTES.rst [hunk 2]
```diff
@@ -18,6 +18,293 @@
...
+User Interface
+~~~~~~~~~~~~~~
+
+-  Updates the help link of the homepage to the actual help instead of
+   the old GitHub wiki page.
+
+Circulation
+~~~~~~~~~~~
+
+-  Fixes ILL request form validation issues that prevents ILL request to
```
Comment (mmo) [inline RELEASE-NOTES.rst:36]:
```suggestion
-  Fixes ILL request form validation issues that prevents ILL requests to
```

File: RELEASE-NOTES.rst [hunk 3]
```diff
@@ -18,6 +18,293 @@
...
+
+Professional interface
+^^^^^^^^^^^^^^^^^^^^^^
+
+-  Adds a new resource to allow the librarian and the system librarian
+   to create templates for several other resources such as document,
+   holdings, item, patron. The template JSON has a non controlled filed,
+   named data, that contains the pre-validated data.
+   A template can be private and thus available for use only to its
+   creator, or public and available to all librarian of the related
```
Comment (mmo) [inline RELEASE-NOTES.rst:114]:
```suggestion
   creator, or public and available to all librarians of the related
```

File: RELEASE-NOTES.rst [hunk 4]
```diff
@@ -18,6 +18,293 @@
...
+   organisation. Only system librarian can create or edit public
```
Comment (mmo) [inline RELEASE-NOTES.rst:115]:
```suggestion
   organisation. Only system librarians can create or edit public
```

File: RELEASE-NOTES.rst [hunk 5]
```diff
@@ -18,6 +18,293 @@
...
+-  Allows to group menu entries in the menu blocks of the professional
+   interface homepage.
+-  Creates the template brief view (search results) and detailed view.
+-  Removes the possibility to add a template from the brief view, as it
+   has to be done from the relevant resource.
+-  Adapts the document, item, holdings, patron routes in order to be
+   able to load data from an existing template in the editor.
+-  Adds a *use it* functionality through a ``canUse`` permission that
+   allows the user to use a template.
+-  Allows duplication of a records from the record detailed view. This
```
Comment (mmo) [inline RELEASE-NOTES.rst:128]:
```suggestion
-  Allows duplication of records from the record detailed view. This
```

File: RELEASE-NOTES.rst [hunk 6]
```diff
@@ -18,6 +18,293 @@
...
+-  Displays the holdings depending on the ``holding_type`` metadata from
+   the holdings itself, and not on the parent document type.
+-  Renames the *patrons* entry of the *User services* menu into *users*.
+
+Circulation
+~~~~~~~~~~~
+
+-  Fixes the requesting patron last name and surname when a requested
+   item is checked in.
+-  Adds a new resource for ILL request, with related permissions (API)
```
Comment (mmo) [inline RELEASE-NOTES.rst:141]:
```suggestion
-  Adds a new resource for ILL requests, with related permissions (API)
```

File: RELEASE-NOTES.rst [hunk 7]
```diff
@@ -18,6 +18,293 @@
...
+   and tests.
+-  Displays the ILL requests in a new tab of the patron profile.
+-  Adds a new form for ILL requests, available for the patron through
```
Comment (mmo) [inline RELEASE-NOTES.rst:144]:
```suggestion
-  Adds a new form for ILL requests, available to the patron in
```

File: RELEASE-NOTES.rst [hunk 8]
```diff
@@ -18,6 +18,293 @@
...
+   the public interface. This form allows the patron to place a request
+   for a document not available in the patron’s organisation, and to
+   select the pickup library.
+-  Improves the loan API performance to render the patron account
+   (public and Professional) quicker.
```
Comment (mmo) [inline RELEASE-NOTES.rst:149]:
```suggestion
   (public and professional) quicker.
```

File: RELEASE-NOTES.rst [hunk 9]
```diff
@@ -18,6 +18,293 @@
...
+   checkin-checkout: a first call is requested to get linked item pids
+   and barcode, then for each barcode, a second call is done to get the
+   item details.
+
+Metadata
+~~~~~~~~
+
+-  Improves the regular expression used in the JSON schema to validate
+   dates, in all relevant resources.
+-  Create the new “collection” resource, to group item together.
```
Comment (mmo) [inline RELEASE-NOTES.rst:160]:
```suggestion
-  Create the new “collection” resource, to group items together.
```

File: RELEASE-NOTES.rst [hunk 10]
```diff
@@ -18,6 +18,293 @@
...
+   -  ``enumerationAndChronology``.
+   -  ``supplementaryContent``.
+   -  ``index``.
+   -  ``missing_issues``.
+   -  ``notes``.
+
+-  Removes conditionality of the 2nd call number depending on an
+   existing 1st call number.
+-  Extends availability of the 2nd call number to all holdings types.
+-  Prevents deleted serial issue to be displayed on the public
```
Comment (mmo) [inline RELEASE-NOTES.rst:173]:
```suggestion
-  Prevents deleted serial issues to be displayed on the public
```

File: RELEASE-NOTES.rst [hunk 11]
```diff
@@ -18,6 +18,293 @@
...
+   interface.
+-  Harmonizes the use of JSON schema custom options for the editor for
```
Comment (mmo) [inline RELEASE-NOTES.rst:175]:
```suggestion
-  Harmonizes the use of JSON schema custom options in the enumeration editor.
```

File: RELEASE-NOTES.rst [hunk 12]
```diff
@@ -18,6 +18,293 @@
...
+   enumeration.
```
Comment (mmo) [inline RELEASE-NOTES.rst:176]:
```suggestion
```

File: RELEASE-NOTES.rst [hunk 13]
```diff
@@ -18,6 +18,293 @@
...
+
+Data
+~~~~
+
+-  Adds templates records fixtures.
```
Comment (mmo) [inline RELEASE-NOTES.rst:181]:
```suggestion
-  Adds template records fixtures.
```

File: RELEASE-NOTES.rst [hunk 14]
```diff
@@ -18,6 +18,293 @@
...
+-  Fixes ``import_users`` CLI to prevent storing pids in the
+   ``patron_pid`` database table, because it results in issues when
+   creating new user after the initial setup.
```
Comment (mmo) [inline RELEASE-NOTES.rst:184]:
```suggestion
   creating new users after the initial setup.
```

File: RELEASE-NOTES.rst [hunk 15]
```diff
@@ -18,6 +18,293 @@
...
+-  Removes the ``append`` argument from the ``import_users`` function.
+-  Adds a username to the user fixtures.
+
+Search
+~~~~~~
+
+-  Fixes ElasticSearch bulk indexing to improve performance of parallel
+   indexing with MEF authority link creation.
+-  Adds facets to the templates search view, to filter results by
```
Comment (mmo) [inline RELEASE-NOTES.rst:193]:
```suggestion
-  Adds facets to the template search view, to filter results by
```

File: RELEASE-NOTES.rst [hunk 16]
```diff
@@ -18,6 +18,293 @@
...
+   -  Expiration date (displayed in the patron profile).
+   -  Library affiliation.
+
+-  Group the patron data of the user in a nested structure.
+-  Adds a new Invenio account login view REST API.
+-  Moves the link between the patron record and the user profile from
+   the email to the id.
+-  Synchronizes the patron record and RERO ID profile data in both
+   directions.
+-  Sets the default user password as the birth date.
```
Comment (mmo) [inline RELEASE-NOTES.rst:218]: My curiosity: birth date or birth year?
If the former, date format variations may be an issue (error-prone), but I reckon this is a PO discussion.
(FWIW, in my local library it's birth year.)

File: RELEASE-NOTES.rst [hunk 17]
```diff
@@ -18,6 +18,293 @@
...
+-  Renames the patron API endpoints from ``patrons`` to ``rero_users``.
+-  Adds a user web API to return the number of patrons giver a username
```
Comment (mmo) [inline RELEASE-NOTES.rst:220]:
```suggestion
-  Adds a user web API to return the number of patrons given a username
```

File: RELEASE-NOTES.rst [hunk 18]
```diff
@@ -18,6 +18,293 @@
...
+   in order to ensure that the usernames are unique.
```
Comment (mmo) [inline RELEASE-NOTES.rst:221]:
```suggestion
   in order to ensure that usernames are unique.
```

File: RELEASE-NOTES.rst [hunk 19]
```diff
@@ -18,6 +18,293 @@
...
+
+Tests
+~~~~~
+
+-  Adds fixtures data for a new organisation for testing purpose.
```
Comment (mmo) [inline RELEASE-NOTES.rst:226]:
```suggestion
-  Adds fixtures for a new organisation for testing purposes.
```

File: RELEASE-NOTES.rst [hunk 20]
```diff
@@ -18,6 +18,293 @@
...
+-  Tests with Cypress the `circulation scenario B`_.
+-  Moves from Travis CI to GitHub actions to improve the preformance of
+   running tests at each pull request or merged commit.
+-  Updates Cypress tests to the patron module refactoring (renamed
+   ``users``).
+
+Angular application (Professional interface, search)
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+-  Initializes the application according to angular standards.
```
Comment (mmo) [inline RELEASE-NOTES.rst:243]:
```suggestion
-  Initializes the application according to Angular standards.
```

standards angulaires... :smile: I like it!

File: RELEASE-NOTES.rst [hunk 21]
```diff
@@ -18,6 +18,293 @@
...
+   records computation.
+-  Fixes an issue that prevented the Celery configuration fixture to be
+   found by setting the constraint on the celery python package version
+   (``<5.0.0``).
+-  Fixes ``LXML`` errors during deployment. See `this Invenio pull
+   request`_.
+-  Enables ``invenio-admin`` and ``invenio-userprofiles``.
+-  Adds an API to display the database connection counts. It allows to
+   monitor the DB usage and to have statistics in order to decide how to
+   improve performance of deployed instance.
```
Comment (mmo) [inline RELEASE-NOTES.rst:262]:
```suggestion
   improve the performance of a deployed instance.
```

File: RELEASE-NOTES.rst [hunk 22]
```diff
@@ -18,6 +18,293 @@
...
+
+Issues
+~~~~~~
+
+-  `rero-ils#83`_: Types are deprecated in ElasticSearch, then
+   ``document_type`` parameter should not be uses anymore. Fixed by
```
Comment (mmo) [inline RELEASE-NOTES.rst:268]:
```suggestion
   the ``document_type`` parameter should not be used anymore. Fixed by
```

File: RELEASE-NOTES.rst [hunk 23]
```diff
@@ -18,6 +18,293 @@
...
+   upgrading ElasticSearch to ``3.3``.
```
Comment (mmo) [inline RELEASE-NOTES.rst:269]: Funny: a few lines above it's stated *Updates ElasticSearch to* **7.9.0**

File: RELEASE-NOTES.rst [hunk 24]
```diff
@@ -18,6 +18,293 @@
 Release notes
 =============
 
+v0.13.1
+-------
+
+This release note includes the changes of the ``rero-ils-ui`` project
+`link`_.
```
[the remark this answers is not in the document]
Comment (iGormilhit) [reply RELEASE-NOTES.rst:25]: The idea is to provide a link to the project, not to mention a *project link*, thus the brakets.

File: RELEASE-NOTES.rst [hunk 25]
```diff
@@ -18,6 +18,293 @@
...
+   the user personnal data, its credentials.
+-  Allows the user to sign in with the username or the email.
+
+Professional interface
+^^^^^^^^^^^^^^^^^^^^^^
+
+-  Adds a new resource to allow the librarian and the system librarian
+   to create templates for several other resources such as document,
+   holdings, item, patron. The template JSON has a non controlled filed,
+   named data, that contains the pre-validated data.
```
[the remark this answers is not in the document]
Comment (iGormilhit) [reply RELEASE-NOTES.rst:112]: The record contains a file labelled `data`. This field is not validated by the template JSON schema.

File: RELEASE-NOTES.rst (hunk 16 above)
Comment (iGormilhit) [reply RELEASE-NOTES.rst:218]: @mmo Actually, it's birth date. Maybe @jma can tell how he handles this.

File: RELEASE-NOTES.rst (hunk 23 above)
Comment (iGormilhit) [reply RELEASE-NOTES.rst:269]: You're right! Should be Invenio 3.3.

**Review by BadrAly — APPROVED** [review]

**Pull request merged**